optimum/commands/onnxruntime/optimize.py

```diff
@@ -0,0 +1,71 @@
+from argparse import ArgumentParser
+from pathlib import Path
+
+from ...onnxruntime.configuration import AutoOptimizationConfig
+from ...onnxruntime.optimization import ORTOptimizer
+
+
+def parse_args_onnxruntime_optimize(parser):
+    required_group = parser.add_argument_group("Required arguments")
+    required_group.add_argument(
+        "--onnx_model",
+        type=Path,
+        required=True,
+        help="Path to the repository where the ONNX models to optimize are located.",
+    )
+
+    optional_group = parser.add_argument_group("Optional arguments")
+    optional_group.add_argument(
+        "-o",
+        "--output",
```

Comment:
Same comment

Reply:
The output is now optional and it is not possible to make it regular like for `export` because the command takes by default the `onnx_model` folder as default output. Unless this is not the behavior you would like?

Reply:
Let's keep it like that then!